.github/workflows/basic.yml

```
@@ -84,6 +84,8 @@ jobs:
     - name: Run tests
       run: |
         # FIXME: The unit tests currently only work with editable installs
+        # Install DIRACCommon first to ensure dependencies are resolved correctly
+        pip install -e ./dirac-common
```

To be on the safe side ?
```suggestion
        pip install -e ./dirac-common[testing]
```